routing/manager.go

```diff
@@ -182,6 +200,8 @@ type RoutingManager struct {
 	ChDone    chan struct{}
 }
 
+var _ ChannelRouter = (*RoutingManager)(nil)
```

Comment:
Can you please add a rationale comment here (the one I mentioned above a few times)? Thanks!